packages/logger/src/__tests__/index.test.ts

```diff
@@ -0,0 +1,138 @@
+import { PassThrough } from "stream";
+import * as winston from "winston";
+import WinstonTransport from "winston-transport";
+import * as bunyan from "bunyan";
+import * as loglevel from "loglevel";
+import * as log4js from "log4js";
+import type { Logger } from "..";
+
+const LOWEST_LOG_LEVEL = "debug";
+const KNOWN_DEBUG_MESSAGE = "This is a debug message";
+
+// For typechecking the logger in question's compatibility
+// with the Logger interface.
+interface HasLogger {
+  logger: Logger;
+}
+
+describe("Logger interface compatibility", () => {
```

Comment:
I'm a little surprised that we don't test `console` here. Maybe at least test that `console` has the right compile-time type even if we don't want to bother mocking it out for a real test?

Reply:
Good call, added tests for console in https://github.com/apollographql/apollo-utils/commit/911dda3f91623624f3a01515c4a131ebb69411fc